jsRofex.js debería llamarse Rofexjs.js

Cuando quiero usar el módulo tengo un "Error: Cannot find module '/app/node_modules/rofexjs/Rofexjs.js' "

que se soluciona simplemente cambiándole el nombre a jsRofex.js por Rofexjs.js pero en servicios web como Heroku se complica un poco

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "rofexjs",
   "version": "1.1.4",
   "description": "--- # Documentación Rofexjs ---",
-  "main": "Rofexjs.js",
+  "main": "jsRofex.js",
   "dependencies": {
     "request": "^2.88.0",
     "ws": "^7.2.0",

